cmd/scollector/conf/conf.go

```diff
@@ -119,6 +119,7 @@ type ProcessDotNet struct {
 type HTTPUnit struct {
 	TOML  string
 	Hiera string
+	Freq  int
```

Comment:
I'd like this documented. Is this seconds? minutes? something else? cmd/scollector/doc.go should have docs for this.